### ISSUE TITLE
CPU speed incorrect in JSON output

### ISSUE THREAD

**Issue by apertome** [perfsonar/toolkit#247]
Graphical output appears to be correct, and 2x higher

Reported by Jason Z

**Comment by apertome** [comment]
<img width="360" alt="screen shot 2017-05-16 at 7 28 17 am" src="https://cloud.githubusercontent.com/assets/6306575/26114821/53b52db6-3a2c-11e7-89b9-3332965a6db6.png">

**Comment by apertome** [comment]
<img width="303" alt="screen shot 2017-05-16 at 7 28 22 am" src="https://cloud.githubusercontent.com/assets/6306575/26114837/5bd5532c-3a2c-11e7-992c-3a89b64edff3.png">

**Comment by apertome** [comment]
So the problem here is that host either has CPU scaling enabled, or it's a product of the linux kernel cpu speed governance.

And the Toolkit returned the current CPU speed, not the max.

This ended up being more complicated than I expected, as `/proc/cpuinfo` and `lscpu` don't gives max CPU speed in most cases. We were able to find one case where an AMD processor had max speed specified.

I updated this so that it attempts to detect the max cpu speed, but falls back to showing current speed if that's not possible.
